frontend/src/features/comparisonSeries/ComparisonSeries.tsx

```diff
@@ -16,6 +26,14 @@ interface Props {
   length: number;
 }
 
+export function getUserComparisons(
+  pollName: string
+): Promise<ComparisonModel[]> {
+  return UsersService.usersMeComparisonsList({
+    pollName,
+  }).then((data) => data.results ?? []);
```

Comment:
I am quite convince that it makes the code simpler and less error prone to use `async` and `await` whenever needed while avoiding `Promises` and `then` as much as possible. As of now we have a bit of a mix of both styles which is probably worse than either one?

Reply:
Yes you're right. I also remember you made this suggestion a long time ago in another pull request.

I'll see what I can do.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/705/commits/6ce3adc9dc6d6e8fce008dcf1a988cec1778393c.